documentation-website/Writerside/topics/DAO-CRUD-Operations.topic

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic
+        SYSTEM "https://resources.jetbrains.com/writerside/1.0/xhtml-entities.dtd">
+<topic xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/writerside/1.0/topic.v2.xsd"
+       title="CRUD Operations" id="DAO-CRUD-Operations">
+    <show-structure for="chapter,procedure" depth="2"/>
+
+    <p>
+        CRUD (Create, Read, Update, Delete) are the four basic operations supported by any database. This section
+        demonstrates how to perform SQL CRUD operations using Exposed's DAO (Data Access Object) API.
+    </p>
+    <chapter title="Create" id="create">
+        <p>
+            To create a new table row, use the <code>new</code> query.
+        </p>
+        <code-block lang="kotlin">
+            val movie = StarWarsFilm.new {
+                name = "The Last Jedi"
+                sequelId = 8
+                director = "Rian Johnson"
+            }
+        </code-block>
```

Comment:
Since this is now split from the original single page, there's no obvious reference to what `StarWarsFilm` is or to what table it is associated. Should there maybe be a link to DAO-Table-Types page at the top of this new page, so readers know where to look for the definition of tables/entities used in the code blocks?

Reply:
I clarified the example a little bit and added a link to the 'Defining entity instances' section, but I'll take a note to also expand this section and improve the samples.